### PR TITLE
Add new company - Fiverr

### PR DESCRIPTION
**Pull request by FelipeFama**
This Pull Request should adhere to the repository's [Code of Conduct](../../main/.github/CONTRIBUTING.md).

Title your pull request descriptively. `[edit X company]` or `[add new company - COMPANY]` 

Please read and check the boxes in the list below that apply!

If updating or adding a company, please include details here: _delete this and add details_

Check off this list as appropriate (add an `x` between the `[]`). It will be used by @dougaitken when reviewing your contribution.

- [] This PR contains housekeeping only (URL edits, copy changes etc) - **if this is checked, delete other lines that don't apply.**
- [] You are an employee of the company mentioned and confirm all included details are correct.
- [] You know your alphabet - company is listed in alphabetical order in the README.md file.
- [] The company directly hires employees. ** No bootcamps / freelance sites / etc**
- [] The company hires remote employees, or positions are available to remote workers and are clearly illustrated as such.
- [] A [company profile](/company-profiles/example.md) is included - This is __required__ for new additions.
- [] An entry has been added or edited within the [readme.md file](../../main/README.md).
- [] __Remote status__ has details regarding how the culture includes remote employees, how the company integrated remote workers, etc
- [] __Region__ details the geographic regions in which this company employees can reside. For more details see the instructions in the [example company profile](/company-profiles/example.md#region).
- [] __How to apply__ details the best approach for new applications, page on site where open position are listed, and any other help available for job hunters

This is a modified version of the [Contributing Guidelines](../../main/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by KieranRobson** [comment]
See @Christian-Oleson comment on your other pull request as the same issues arise here: https://github.com/remoteintech/remote-jobs/pull/1539#pullrequestreview-1143229329

File: README.md [hunk 1]
```diff
@@ -236,6 +236,7 @@ Name | Website | Region
 [Fire Engine Red](/company-profiles/fire-engine-red.md) | https://fire-engine-red.com/ | Worldwide
 [Fireball Labs](/company-profiles/fireball-labs.md) | https://www.fireballlabs.com | Germany
 [FivexL](/company-profiles/fivexl.md) | https://fivexl.io | Worldwide
+[Fiverr](/company-profiles/fiverr.md) | https://www.fiverr.com/ | North America, Asia
```
Comment (KieranRobson) [inline README.md:239]: This should be above `Fivexl`

Comment (FelipeFama) [reply README.md:238]: okay

**Review by dougaitken — CHANGES_REQUESTED** [review]
Please add region section back to the profile so it matches the regions mentioned in the readme

File: company-profiles/fiverr.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+## Company size
+
+201 - 500
+
+## Remote status
+
+Work in the place that makes you happy, that inspires you daily and that helps you become the person you want to be. We were born with 100% 
+remote teams and we always will be.
+
+## Office Locations
```
Comment (KieranRobson) [inline company-profiles/fiverr.md:16]: Quick change! 

`Office Locations` to `Office locations`

**Review by dougaitken — APPROVED** [review]

**Comment by dougaitken** [comment]
Thanks @FelipeFama

**Pull request merged**